src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/package.json

```diff
@@ -12,42 +12,45 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "6.1.10",
-    "@angular/common": "6.1.10",
-    "@angular/compiler": "6.1.10",
-    "@angular/core": "6.1.10",
-    "@angular/forms": "6.1.10",
-    "@angular/http": "6.1.10",
-    "@angular/platform-browser": "6.1.10",
-    "@angular/platform-browser-dynamic": "6.1.10",
-    "@angular/platform-server": "6.1.10",
-    "@angular/router": "6.1.10",
-    "@nguniversal/module-map-ngfactory-loader": "6.0.0",
-    "core-js": "^2.5.4",
-    "rxjs": "^6.0.0",
-    "zone.js": "^0.8.26",
+    "@angular/animations": "7.2.5",
+    "@angular/common": "7.2.5",
+    "@angular/compiler": "7.2.5",
+    "@angular/core": "7.2.5",
+    "@angular/forms": "7.2.5",
+    "@angular/http": "7.2.5",
+    "@angular/platform-browser": "7.2.5",
+    "@angular/platform-browser-dynamic": "7.2.5",
+    "@angular/platform-server": "7.2.5",
+    "@angular/router": "7.2.5",
+    "@nguniversal/module-map-ngfactory-loader": "7.1.0",
+    "core-js": "^2.6.5",
+    "rxjs": "^6.4.0",
+    "zone.js": "^0.8.29",
     "aspnet-prerendering": "^3.0.1",
-    "bootstrap": "^4.1.3",
+    "bootstrap": "^4.3.1",
     "jquery": "3.3.1",
+////#if (IndividualLocalAuth)
```

Comment:
Wasn't aware we could conditionalize these, that's good to know.

Reply:
I suspect you can't realistically do this, without potentially hundreds of conditionals in `package-lock.json`.

Reply:
There is an alternative, you can generate the lock with and without and at template instantiation time rename the lock files to be the final thing.
For example packages.lock.json.IndividualAuth and packages.lock.json.None and at instantiation time you rename them to packages.lock.json we do that with the EF migrations.

Reply:
But i'm not doing this today.